bouncer/tests/lp_api_test.ts

```diff
@@ -21,7 +21,7 @@ import { getChainflipApi, observeEvent } from '../shared/utils/substrate';
 import { ExecutableTest } from '../shared/executable_test';
 
 /* eslint-disable @typescript-eslint/no-use-before-define */
-export const testLpApi = new ExecutableTest('LP-API', main, 200);
+export const testLpApi = new ExecutableTest('LP-API', main, 1000);
```

Comment:
I also increased the timeout of this test - it contains multiple other tests with a 500 second timeout. We don't want the timeout of the parent to be stricter.

Reply:
Actually the 200 sec timeout was correct. I just set the child tests timeouts to 500 so they would never get hit and could be ignored. Was easier than setting correct timeouts for each.

Reply:
Surely it's better to allow the child timeouts to be hit? This way we get more granular feedback in terms of what went wrong. If we want 200, then let's set all the child timeouts to 200, and the parent to 500 or something.

Reply:
Looks like you updated it - much better, thanks.